diff --git a/src/steuerung3d/apps/yellow/engines/hip/engine.py b/src/steuerung3d/apps/yellow/engines/hip/engine.py
--- a/src/steuerung3d/apps/yellow/engines/hip/engine.py
+++ b/src/steuerung3d/apps/yellow/engines/hip/engine.py
@@ -1,6 +1,6 @@
 from .types import (
     HipAttachInputs,
     HipBannerInputs,
     HipParamButtons,
     HipParamGroup,
     HipParamUiState,
     HipState,
     HipStepInputs,
     HipStepResult,
+    HipUiInputs,
 )
